qtop.py

```diff
@@ -47,20 +51,23 @@
 #     options.COLORFILE = os.path.expanduser('~/qtop/qtop/qtop.colormap')
 
 
-def colorize(text, pattern, color=None):
+def colorize(text, pattern='Nothing', color_func=None, bg_colour=None):
     """
     prints text coloured according to a unix account pattern color.
+    If color is given, pattern is not needed.
     """
+    # bg_colour = code_of_color['BlueBG']
+    bg_colour = '' if not bg_colour else bg_colour
     try:
-        color_code = code_of_color[color] if color else code_of_color[color_of_account[pattern]]
+        ansi_color = code_of_color[color_func] if color_func else code_of_color[color_of_account[pattern]]
```

Comment:
for now it's a dummy with no purpose. This will be made in modularisation part two